common/filestore_utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for interacting with the file storage."""
+
+from common import logs
+from common import experiment_utils
+
+logger = logs.Logger('filestore_utils')
+
+
+def _using_gsutil():
+    """ Test whether using gsutil format paths. """
```

Comment:
Change comment to `Returns True if using Google Cloud Storage for filestore`.
Also, don't put an extra space at the beginning and end of each docstring.